sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -20,7 +20,6 @@
 
 if TYPE_CHECKING:
     from azure.core.credentials import TokenCredential
-    from ._common.auto_lock_renewer import AutoLockRenewer
```

Comment:
This doesn't cause typechecking issues?

Reply:
no typechecking issue as AutoLockRenewer is in kwargs, it's the sphinx docstring issue that abs ref fixed.